### PR TITLE
[11.0] [MIG] mig account fiscal position rule purchase

### PR DESCRIPTION
**Pull request by Tonow-c2c**
Need review
Need this module before [https://github.com/OCA/account-fiscal-rule/pull/78](https://github.com/OCA/account-fiscal-rule/pull/78)

### CODE REVIEW

File: account_fiscal_position_rule_purchase/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Copyright (C) 2009-TODAY Akretion <http://www.akretion.com>
```
Comment (leemannd) [inline account_fiscal_position_rule_purchase/__init__.py]: Copyright can be dropped from `init.py` files

File: account_fiscal_position_rule_purchase/tests/test_account_fiscal_position_rule_purchase.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
+# Copyright 2018 Camptocamp SA
+#   @author: Thomas Nowicki
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp.tests.common import TransactionCase
```
Comment (leemannd) [inline account_fiscal_position_rule_purchase/tests/test_account_fiscal_position_rule_purchase.py]: s/openerp/odoo/
Do you have runned tests locally? It doesn't fail with the import from `openerp` ?

**Review by leemannd — CHANGES_REQUESTED** [review]
A few changes are required

File: account_fiscal_position_rule_purchase/security/ir.model.access.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+
```
Comment (ghost) [inline account_fiscal_position_rule_purchase/security/ir.model.access.csv]: Is it is a reason to add an "empty" file?

**Review by ghost — APPROVED** [review]
LGTM

**Review by ghost — COMMENTED** [review]
Please fix oca_dependencies https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/oca_dependencies.txt

**Comment by Tonow-c2c** [comment]
No need for this module cause he depend of any other OCA module

**Comment by dreispt** [comment]
Superseded by #107

**Pull request closed without merging**